src/ScopeProvider/index.ts

```diff
@@ -0,0 +1 @@
+export { ScopeContext, ScopeProvider } from './ScopeProvider';
```

Comment:
I don't think we need this file. Just import it directly from the library index. Though, it's just a preference.

Reply:
done.

I also added exporting INTERNAL_ScopeContext from `src/ScopeProvider/ScopeProvider.tsx`, but am intentionally excluding this export from the package. I hope this will be a good compromise.

Reply:
Thanks.

As the current config in this repo bundles `dist`, exporting only in the source file doesn't work.
Anyway, let's drop it: https://github.com/jotaijs/jotai-scope/pull/33#discussion_r1607469134